services/core/java/com/android/server/lights/LightsService.java

```diff
@@ -55,6 +57,12 @@ private LightImpl(Context context, int id) {
                 }
             }
             mSurfaceControlMaximumBrightness = maximumBrightness;
+            try {
+                mUseScaleBrightness = context.getResources()
+                    .getInteger(com.android.internal.R.integer.config_useScaleBrightness);
+            } catch (Resources.NotFoundException e) {
```

Comment:
A "better safe than sorry" try catch?

Reply:
> A "better safe than sorry" try catch?

At previous PR, you're too afraid of being derped (even if will never, since it already defined on fwb's aosip_config.xml).

Reply:
Yea, If already defined (and that is at all times). No need for a try catch imo

Reply:
I will merge and amend to remove the try catch if that's fine with you